app/models/orphan.rb

```diff
@@ -66,6 +66,8 @@ class Orphan < ActiveRecord::Base
   accepts_nested_attributes_for :current_address, allow_destroy: true
   accepts_nested_attributes_for :original_address, allow_destroy: true
 
+  default_scope { includes(:partner, :orphan_sponsorship_status, :orphan_status, original_address: :province) }
```

Comment:
Is this too much eager loading for the default scope? Every `find` etc. query will load up 5 AR objects in addition to the orphan itself. Should this be limited to the orphans controller?

Somewhat related - switching statuses from associations to `enums` will boost performance in addition to other benefits. Is it time yet?